### PR TITLE
logs: Improve "mempool min fee not met" error

### PR DESCRIPTION
**Pull request by Varunram**
Intended as a fix for #11955. 
I'm pretty new here, so apologies if this isn't the intended fix. Thanks!

### CODE REVIEW

**Comment by laanwj** [comment]
nFees, mempoolRejectFee is already in the extended description (strDebugMessageIn) - A broader fix would be to log/return that where it's reported, instead of changing the main message.

**Comment by Varunram** [comment]
Thanks, I'll do the changes and update the PR

File: src/wallet/fees.cpp [hunk 1]
```diff
@@ -59,6 +59,8 @@ CAmount GetMinimumFee(unsigned int nTxBytes, const CCoinControl& coin_control, c
         if (fee_needed < min_mempool_fee) {
             fee_needed = min_mempool_fee;
             if (feeCalc) feeCalc->reason = FeeReason::MEMPOOL_MIN;
+        } else {
+            LogPrintf("Minimum mempool fee(%d) not met", min_mempool_fee);
```
Comment (laanwj) [inline src/wallet/fees.cpp]: I don't think `Get*` is a good place to log a message.

What I meant is that in the current place where you see "mempool min fee not met" being logged, it should log the debug message of the state too.

Comment (Varunram) [reply src/wallet/fees.cpp]: Thanks for the review, I'll do the changes.

**Comment by fanquake** [comment]
Closing for now. Please reopen when you've adjusted this PR.

**Pull request closed without merging**